nslsii/tests/test_ipynb.py

```diff
@@ -6,3 +6,8 @@ def test_ipynb():
     obj.metadata
     obj.url
     obj.reload()
+
+def test_touchbl():
+    from nslsii.common import if_touch_beamline
+    if if_touch_beamline():
+        pass
```

Comment:
Let's use assert to test that if the `TOUCHBEAMLINE` env var is not set, the function returns `False`, otherwise - `True`. The env var can be set via `os.environ["TOUCHBEAMLINE"] = ...` for testing inside of the test. We can test multiple inputs as well via `@pytest.mark.parametrize`.